react/components/UtilityFunctionsPixexEvents.js

```diff
@@ -0,0 +1,21 @@
+export const getCategoryFromObjs = (products) => {
+  if (products[0] === null || products[0] === undefined) {
+    return ''
+  }
+
+  const { categoryId } = products[0]
+  const result = products.every((product) => product.categoryId === categoryId)
+
+  return result ? categoryId : ''
+}
+
+export const pushPixelEvent = (name, value, products, push) => {
+  push({
+    event: 'filterManipulation',
+    items: {
+      filterProductCategory: getCategoryFromObjs(products),
+      filterName: name,
+      filterValue: value,
+    },
+  })
+}
```

Comment:
I would rename the file name from UtilityFunctions**Pixex**Events to `pixelEvents`, if the pattern is pascal case: PixelEvents, what do you think @igorbrasileiro @icazevedo ?

Reply:
I agree! This would be a better naming. Maybe specify it related to filter manipulation? For instance: `filterManipulationPixelEvents`

Reply:
File name changed